en/reference/annotations-mapping.rst

```diff
@@ -49,6 +49,10 @@ Document
 Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
+-  **uniqueNodeType**: If this document uses a unique node type, set to ``true``
+   in order to support outer joins correctly; unnecessary in all other cases. See
```

Comment:
lets not say "unnecessary in all other cases", i fear we will forget it there even if we use it in the future to make all joins more efficient.